platform/ios/src/MGLAPIClient.m

```diff
@@ -106,7 +107,8 @@ - (void)setupUserAgent {
     NSString *appName = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleName"];
     NSString *appVersion = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleShortVersionString"];
     NSString *appBuildNumber = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleVersion"];
-    _userAgent = [NSString stringWithFormat:@"%@/%@/%@ %@", appName, appVersion, appBuildNumber, MGLAPIClientUserAgent];
+    NSString *sdkVersion = [NSBundle mgl_frameworkBundle].infoDictionary[@"CFBundleShortVersionString"];
```

Comment:
I think that would be very useful @1ec5 

@camilleanne @mick would that be ok? For our prerelease builds, it would mean the UA would look something like `Mapbox GL/0.0.2/7877 MapboxEventsiOS/3.2.0-pre.1`, ``Mapbox GL/0.0.2/7877 MapboxEventsiOS/3.2.0-pre.2`, etc..